oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/WellknownHandler.java

```diff
@@ -104,6 +105,14 @@ public APIGatewayProxyResponseEvent handleRequest(
                                 providerMetadata.setServiceDocsURI(new URI("http://TBA"));
                                 providerMetadata.setEndSessionEndpointURI(
                                         buildURI("/logout", baseUrl));
+                                providerMetadata
+                                        .toJSONObject()
+                                        .appendField(
+                                                "TrustMarkHandler",
```

Comment:
so the key should be "trustmarks" and the value is the api endpoint for trustmarks you have just created